tests/Vue/Pages/Home.spec.js

```diff
@@ -18,21 +18,17 @@ describe('Home.vue', () => {
     const localVue = createLocalVue();
     localVue.use(Vuex);
 
-    const store = new Vuex.Store();
-
     it('sanitises input with empty lines', async () => {
-        const formData = { itemsInput: 'Q1\n\nQ2\n' };
+
+        const store = new Vuex.Store({state: { lastSearchedIds: 'Q1\n\nQ2\n' }});
 
         const wrapper = mount(Home, { 
             mocks,
             localVue,
-            store,
-            data() {
-                return { form: formData };
-            },
+            store
         });
 
-        expect( wrapper.vm.serializeInput() ).toEqual('Q1|Q2');
+        expect( wrapper.vm.serializeInputUrl() ).toEqual('Q1|Q2');
```

Comment:
This can probably remain if you remove the new `serializeInputText` method. The name also wouldn't make much sense, since it's not the URL that is serialized, but only the items ids.

Reply:
Adressed with the removal of the serializeInputText commit